guide/popular-topics/webhooks.md

```diff
@@ -125,7 +125,7 @@ webhookClient.send({
 });
 ```
 
-Example using a Webhook:
+To make sure the resulting webhook can be executed by your bot, you need to find a webhook that is managable by your bot. To do so, you can check if a token of the webhook exists.
```

Comment:
In my opinion this sounds quite clunky and doesn't read all too well.

> Try to find a webhook your bot knows the token for. This makes sure your bot can execute the webhook lateron.

I find much easier to read, as an example. Feel free to rephrase or use a different way to phrase it altogether, of course.

Reply:
So for example:
```diff
- To make sure the resulting webhook can be executed by your bot, you need to find a webhook that is manageable by your bot. To do so, you can check if a token of the webhook exists.
+ Try to find a webhook your bot knows the token for. This makes sure your bot can execute the webhook later on.
```
Like that? I don't want to make an unnecessary commit.

Reply:
sure, if you want to rephrase it, feel free

Reply:
Ok, I will do so.